tox.ini

```diff
@@ -43,6 +43,9 @@ commands = flake8 scapy/
 
 # flake8 configuration
 [flake8]
-select = E23,E30,E20,E12,W29,E25,E26
+disabel-noqa = True
```

Comment:
Seems wrong:

> If we ever want to disable Flake8 respecting # noqa comments, we can can refer to flake8 --disable-noqa.

Shoudln't it be disab**le**-noqa (not disab**el**-noqa)

Reply:
Agreed, you should remove this line so that your noqa comments work.